prow/git/v2/remote_test.go

```diff
@@ -225,3 +225,40 @@ func TestHTTPResolverFactory(t *testing.T) {
 		}
 	}
 }
+
+func TestCloneURIFromOrgRepo(t *testing.T) {
```

Comment:
this is WAI, this will never be worked with GitHub repos, this function tested here is only used by the resolver that would ever be initiated with Gerrit repos only. The names of this functions are not super clear, will clean up in followup PR